dataland-community-manager/src/main/resources/email-templates/claim_ownership.html.ftl

```diff
@@ -4,6 +4,8 @@
 <html>
```

Comment:
Reintegrate the senders and receivers evaluation process with "exit" statement if an EmailBuilder has invalid receivers or cc-receivers.

Reply:
Should exist already